stingray/lightcurve.py

```diff
@@ -451,8 +451,8 @@ def join(self, other):
             except IndexError:
                 count2 = None
 
-            if not count1 is None:
-                if not count2 is None:
+            if count1 is not None:
+                if count2 is not None:
```

Comment:
Yes the original way also works, but PEP8 says:

```
Yes:

if foo is not None:
No:

if not foo is None:
```
[Source](https://www.python.org/dev/peps/pep-0008/#programming-recommendations)

:wink: